lib/apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -37,12 +37,12 @@ type handler interface {
 	Configure(api *operations.PortLayerAPI, handlerCtx *handlers.HandlerContext)
 }
 
-var portlayerhandlers = []handler{
-	&handlers.StorageHandlersImpl{},
-	&handlers.MiscHandlersImpl{},
-	&handlers.ScopesHandlersImpl{},
-	&handlers.AttachHandlersImpl{},
-	&handlers.ContainersHandlersImpl{},
+var portlayerhandlers = map[string]handler{
```

Comment:
Sure, it compiles.  It's how you initialize a golang map.  You're right.  It was used earlier when we were using a custom middleware and needed the map.  Reverting.